galaxy-app/app/install_node.sh

```diff
@@ -16,6 +16,13 @@ echo "Installing node version ${NODE_VERSION} to /node-v${NODE_VERSION}-linux-x6
 
 set -x
 
-curl -sSLO "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.gz" \
-  && tar -xzf "node-v$NODE_VERSION-linux-x64.tar.gz" -C / \
-  && rm "node-v$NODE_VERSION-linux-x64.tar.gz"
+if [ "$NODE_VERSION" == "8.11.1" ]
```

Comment:
Maybe we should preinstall this one into the base image (by adding an install_node.sh invocation to the Dockerfile)? Esp since this isn't coming from a CDN.  Hopefully we won't be triggering the npm bug described in the Dockerfile.